.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [["es2015", {"loose": true}]],
+  "plugins": [
```

Comment:
I think we ought to stick with what the generator v2 provides for now. I'd hate to hit some edge case here.